react/react1/week3/todo-app/src/App.css

```diff
@@ -0,0 +1,20 @@
+.App {
+  text-align: center;
+}
+
+li {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  border: solid black 2px;
+  width: 80%;
+  margin: 6px;
+}
+ul {
```

Comment:
Styling HTML elements directly like `li` and `ul` is not a good practice. Try using classnames/ids instead.